rust-runtime/aws-smithy-runtime-api/src/client/interceptors/context/wrappers.rs

```diff
@@ -206,6 +198,14 @@ impl<'a, I, O, E> AfterDeserializationInterceptorContextRef<'a, I, O, E> {
     pub fn output_or_error(&self) -> Result<&O, &OrchestratorError<E>> {
         expect!(self, output_or_error)
     }
+
+    #[doc(hidden)]
+    /// Downgrade this helper struct, returning the underlying InterceptorContext. There's no good
+    /// reason to use this unless you're writing tests or you have to interact with an API that
+    /// doesn't support the helper structs.
+    pub fn inner(&self) -> &'_ InterceptorContext<I, O, E> {
+        self.inner
+    }
```

Comment:
clearly we actually need this so maybe it shouldn't be doc hidden?

Reply:
Do you think we should add it to all the interceptor context wrappers then?

Reply:
Went ahead and added it to all of them.